src/app/navigation/navigation.component.html

```diff
@@ -52,8 +52,9 @@
                   <a class="nav-link" routerLink="about" mdbWavesEffect>About</a>
               </li>
               <li class="nav-item">
-                  <a class="nav-link" href="" mdbWavesEffect>How To Use</a>
-              </li>
+                <a class="nav-link" routerLink="how-to" mdbWavesEffect>How To Use</a>
```

Comment:
this will need to change also to "howto" it has to match what you put in the router module